src/module.ts

```diff
@@ -3,11 +3,13 @@ import { NullifyDataSource } from './datasource';
 import { ConfigEditor } from './components/ConfigEditor';
 import { QueryEditor } from './components/Query/QueryEditor';
 import { NullifyDataSourceOptions, NullifyQueryOptions } from './types';
+import { VariableQueryEditor } from 'components/VariableQueryEditor';
 
 export const plugin = new DataSourcePlugin<
   NullifyDataSource,
   NullifyQueryOptions,
   NullifyDataSourceOptions
 >(NullifyDataSource)
   .setConfigEditor(ConfigEditor)
-  .setQueryEditor(QueryEditor);
+  .setQueryEditor(QueryEditor)
+  .setVariableQueryEditor(VariableQueryEditor);
```

Comment:
Seems like the suggested classes are still marked as "@alpha -- experimental". Will investigate later.